cheatle/__main__.py

```diff
@@ -1,2 +1,24 @@
+from curses.ascii import isalpha
+from email.policy import default
+import click
+
+#function that takes in the known/green values
+#would like to set a default value to "_____" but when I do that within the click option "Green Letters:  [_____]:" is displayed as the prompt
+@click.command()
+@click.option('--known', prompt = "Green Letters", help='Letters that are in the correct position. Use a "_" for letters that are not in the right position')
+def known_word(known):
```

Comment:
The method name should be the command name, not the name of the first option (--known). Let's name this method `cheatle` instead.

Reply:
Done